test/integration/userStatus.test.js

```diff
@@ -75,14 +91,14 @@ describe("UserStatus", function () {
           expect(res.body).to.be.a("object");
           expect(res.body.message).to.equal("User Status found successfully.");
           expect(res.body.userId).to.equal(userId);
-          expect(res.body).to.have.property("monthlyHours");
-          expect(res.body).to.have.property("currentStatus");
+          expect(res.body.data).to.have.property("monthlyHours");
+          expect(res.body.data).to.have.property("currentStatus");
```

Comment:
the test does not tell me that the data returned is for self. it can return for any person if I pass the cookie. lets say i pass in your cookie then i can get your details.

Reply:
since the route is `/self`. we take the userId from req.userData hence it will always be the user status data for the person who has requested it.